pyaedt/modules/PostProcessor.py

```diff
@@ -4911,3 +4915,455 @@ def sample_ami_waveform(
+        return out_dict
+
+    @pyaedt_function_handler()
+    def export_csv(self, filename, sweep_name=None, design_variation={}, intrinsic_value=""):
+        if not sweep_name:
+            sweep_name = self._app.nominal_sweep
+        dv_string = ""
+        for el in design_variation:
+            dv_string += el + "='" + design_variation[el] + "' "
+        self._create_field_summary(sweep_name, dv_string)
+        self._app.osolution.ExportFieldsSummary(
+            [
+                "SolutionName:=",
+                sweep_name,
+                "DesignVariationKey:=",
+                dv_string,
+                "ExportFileName:=",
+                filename,
+                "IntrinsicValue:=",
+                intrinsic_value,
+            ]
+        )
+        return True
+
+    @pyaedt_function_handler()
+    def _create_field_summary(self, setup, variation):
+        arg = ["SolutionName:=", setup, "Variation:=", variation]
+        for i in self.calculations:
+            arg.append("Calculation:=")
+            arg.append(i)
+        self._app.osolution.EditFieldsSummarySetting(arg)
+
+
+class IcepakPostProcessor(PostProcessor, object):
+    def __init__(self, app):
+        PostProcessorCommon.__init__(self, app)
+
+    @pyaedt_function_handler()
+    def create_field_summary(self):
+        return FieldSummary(self._app)
+
+    @pyaedt_function_handler()
+    def get_fans_operating_point(self, export_file=None, setup_name=None, timestep=None, design_variation=None):
+        """
+        Get the operating point of the fans in the design.
+
+        Parameters
+        ----------
+        export_file : str, optional
+            Name of the file to save the operating point of the fans. The default is
+            ``None``, in which case the filename is automatically generated.
+        setup_name : str, optional
+            Setup name to determine the operating point of the fans. The default is
+            ``None``, in which case the first available setup is used.
+        timestep : str, optional
+            Time, with units, at which to determine the operating point of the fans. The default
+            is ``None``, in which case the first available timestep is used. This parameter is
+            only relevant in transient simulations.
+        design_variation : str, optional
+            Design variation to determine the operating point of the fans from. The default is
+            ``None``, in which case the nominal variation is used.
+
+        Returns
+        -------
+        list
+            First element of the list is the CSV filename. The second and third elements
+            are the quantities with units describing the operating point of the fans.
+            The fourth element is a dictionary with the names of the fan instances
+            as keys and lists with volumetric flow rates and pressure rise floats associated
+            with the operating points as values.
```

Comment:
```suggestion
            with the operating point as values.
```
Again--based on earlier descriptions--it seems there is only one operating point for all fans.